### DISCUSSION TITLE
Effect and Xstate

### DISCUSSION THREAD

**Discussion by christophe-g** [statelyai/xstate#4767] [Q&A] [answered]
Hey all, 

I have seen (I think on one of the recent blog post) that @davidkpiano would give a talk on [effect](https://effect.website/) + xstate. 

I would be quite interested to have some examples of how the two play together, but could not find much online. 

Is there anything that can be shared ? 

Thanks a lot.

**Comment by davidkpiano** [comment, marked as answer]
Hey @christophe-g, the first place I would point you to is this excellent article: https://www.sandromaglione.com/articles/getting-started-with-xstate-and-effect-audio-player#implement-actions-using-effect

We'll definitely have more resources soon 📚

**Reply by christophe-g** [reply]
Thanks a lot !

**Reply by PaulRBerg** [reply]
Hey @davidkpiano, did you ever end up writing more about xState + Effect-ts?

**Reply by davidkpiano** [reply]
I haven't had a chance to yet, unfortunately.
